daemon/cmd/daemon_main.go

```diff
@@ -1989,6 +1989,19 @@ func startDaemon(d *Daemon, restoredEndpoints *endpointRestoreState, cleaner *da
 	bootstrapStats.updateMetrics()
 	go d.launchHubble()
 
+	// Start controller to validate daemon config is unchanged
+	cfgGroup := controller.NewGroup("daemon-validate-config")
+	d.controllers.UpdateController(
+		cfgGroup.Name,
+		controller.ControllerParams{
+			Group:  cfgGroup,
+			DoFunc: option.Config.ValidateUnchanged,
```

Comment:
Can we pass a cell.Health for the `ControllerParams.Health` parameters so we have health reporting for this reconciliation loop?